libraries/chain/include/eosio/chain/hotstuff/hotstuff.hpp

```diff
@@ -60,6 +61,10 @@ namespace eosio::chain {
    struct quorum_certificate {
       uint32_t                 block_num;
       valid_quorum_certificate qc;
+
+      qc_claim_t to_qc_claim() const {
```

Comment:
I'm not crazy about including `finality_core.hpp` just to define this function, but it would be equally awkward to include `hotstuff.hpp` into `finality_core.hpp`, so I don't really have a better solution.

Reply:
This file should be eventually renamed and relocated to the chain include directory.